sql/src/test/java/io/crate/analyze/SetAnalyzerTest.java

```diff
@@ -60,6 +60,10 @@ public void testSet() throws Exception {
         assertThat(analysis.isPersistent(), is(true));
         assertThat(analysis.settings().toDelimitedString(','), is("stats.operations_log_size=1,"));
 
+        analysis = analyze("SET GLOBAL TRANSIENT stats.jobs_log_size='1'");
```

Comment:
You can use the test below with multiple assignments to test '1' and 1 and remove this.